README.md

```diff
@@ -122,7 +153,7 @@ For a Websocket connection there is no limit to the number of requests sent down
 
 For rest the base limit per-user is 1,000 orders per 5 minute interval, and is shared between all account API connections. It increases proportionally to your trade volume based on the following formula:
 
-1000 + (TOTAL_PAIRS_PLATFORM * 60 * 5) / (250000000 / USER_VOL_LAST_30d)
+1000 + (TOTAL*PAIRS_PLATFORM * 60 \_ 5) / (250000000 / USER_VOL_LAST_30d)
```

Comment:
**NOTE**
This should possibly be reverted as it seems to have been caused by some formatting. 
I believe it should still be:
```
1000 + (TOTAL_PAIRS_PLATFORM * 60 * 5) / (250000000 / USER_VOL_LAST_30d)
```

Reply:
@itsdeka I've now reverted this typo in the README commit: [f285997](https://github.com/bitfinexcom/bitfinex-api-py/pull/194/commits/f285997c6787c7300261a6f3681ceb08152fee5a)